src/main/io/gps.h

```diff
@@ -132,11 +286,41 @@ typedef struct gpsSolutionData_s {
     uint32_t time;                  // GPS msToW
 } gpsSolutionData_t;
 
+typedef struct ubxVersion_s {
+    uint8_t major;
+    uint8_t minor;
+} ubxVersion_t;
+
+typedef struct ubxSwVersion_s {
+    ubxVersion_t firmwareVersion;
+    ubxVersion_t protocolVersion;
+} ubxSwVersion_t;
+
+/*
+* keeping this table for reference
+typedef enum {
+    UBX_CAP_SAT_NONE = 0x0,
+    UBX_CAP_SAT_GPS = 0x0001,
+    UBX_CAP_SAT_GLO = 0x0002,
+    UBX_CAP_SAT_GAL = 0x0004,
+    UBX_CAP_SAT_BDS = 0x0008,
+    UBX_CAP_SAT_SBAS = 0x0010,
+    UBX_CAP_SAT_QZSS = 0x0020,
+} ubxCapabilities_e;
+*/
+
+typedef struct ubxMonVer_s {
+    ubxSwVersion_t swVersion;
+    uint32_t hwVersion;
+    /*uint32_t extension;*/
```

Comment:
The comment about `uint32_t extension` is unneeded and can be removed. Unless `ubxMonVer_t` has a very specific structure according to the documentation. In this case we should include `uint32_t extension` even if we don't use it.